src/DataTable/DataTable.scss

```diff
@@ -33,6 +34,21 @@ $data-table-footer-position: center !default;
   -webkit-overflow-scrolling: touch;
 }
 
+.pgn__data-table-layout-wrapper{
+  display: flex;
+  align-items: flex-start;
+  .pgn__data-table-layout-sidebar{
+    border-radius: $border-radius;
+    box-shadow: $data-table-box-shadow;
+    padding: $data-table-padding-small;
+    margin-right: $data-table-layout-margin;
```

Comment:
Using one of the predefined spacer variables here might be beneficial to mimic the common utility classes for margins? I.e., `map_get($spacers, 4)`

https://github.com/edx/paragon/blob/a861078c6e4dc452ab580dbdcfdbe46c632fa6c6/scss/core/_variables.scss#L360-L376